packages/Toast/src/Toast.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import CheckIcon from "@paprika/icon/lib/Check";
+import CautionIcon from "@paprika/icon/lib/Caution";
+import ExclamationCircleIcon from "@paprika/icon/lib/ExclamationCircle";
+import InfoCircleIcon from "@paprika/icon/lib/InfoCircle";
+import LockIcon from "@paprika/icon/lib/Lock";
+
+import toastStyles, { contentStyles, IconStyled, CloseButtonStyled } from "./Toast.styles";
+
+const propTypes = {
+  /**
+   * Will make this an "assertive" alert that will interrupt immediately.
+   * Default behaviour is to be "polite" and wait until the user is idle.
+   */
+  ariaAlert: PropTypes.bool,
+
+  /** Will automatically close after 1500ms (or longer if provided by autocloseTimeout) */
+  autoclose: PropTypes.bool,
+
+  /** Duration (in ms) before Toast will automaticall close (if autoclose is true) */
+  autocloseTimeout: PropTypes.number,
+
+  /** Content of the Toast */
+  children: PropTypes.node,
+
+  /** If the component should have a 'close' button */
+  hasCloseButton: PropTypes.bool,
+
+  /** How "controlled" toast is shown / hidden. */
+  isOpen: PropTypes.bool,
+
+  /** Callback that is executed after clicking the 'close' button */
+  onClose: PropTypes.func,
+
+  /** If the Toast is fixed to the top of the viewport */
+  isSticky: PropTypes.bool,
+
+  /** Determines the styling of the Toast */
+  type: PropTypes.oneOf(["success", "warning", "error", "info", "locked", "visually-hidden"]),
+
+  /** The z-index of the Toast */
+  zIndex: PropTypes.number,
+};
+
+const defaultProps = {
+  ariaAlert: false,
+  autoclose: false,
+  autocloseTimeout: 5000,
+  children: null,
+  hasCloseButton: true,
+  isOpen: undefined,
+  onClose: () => {},
+  isSticky: false,
+  type: "info",
+  zIndex: 1006,
```

Comment:
I think we should stick to our 7-layer system for this, but not sure which layer the `<Toast>` should be on by default.  Maybe just `1`?

Reply:
This is the current value we're using in acl-ui, if we change it, some toast might be broken in our apps

Reply:
Only if they refactor to use the Paprika one.  If they do that, they'll have to add their own `zIndex` override... no big deal, in my opinion.